planning/scenario_planning/parking/costmap_generator/CMakeLists.txt

```diff
@@ -10,25 +10,29 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 find_package(ament_cmake_auto REQUIRED)
-find_package(PCL REQUIRED QUIET COMPONENTS common io)
+find_package(PCL REQUIRED COMPONENTS common io)
 ament_auto_find_build_dependencies()
 
 include_directories(
   include
   ${GRID_MAP_INCLUDE_DIR}
   ${PCL_COMMON_INCLUDE_DIRS}
+  ${orocos_kdl_INCLUDE_DIRS}
```

Comment:
I don't think orocos_kdl is used anywhere in the code.
We probably can remove this.

Reply:
@mitsudome-r this is fixed